developer/src/kmc/src/commands/build.ts

```diff
@@ -29,7 +30,19 @@ function commandOptionsToCompilerOptions(options: any): CompilerOptions {
 export function declareBuild(program: Command) {
   BaseOptions.addAll(program
     .command('build [infile...]')
-    .description('Build a source file into a final file')
+    .description(`Compile one or more source files or projects.`)
+    .addHelpText('after', `
+Supported file types:
+  * folder: Keyman project in folder
+  * .kpj: Keyman project
+  * .kmn: Keyman keyboard
+  * .xml: LDML keyboard
+  * .model.ts: Keyman lexical model
+  * .kps: Keyman keyboard package
```

Comment:
Is this also Keyman lexical model package?

Reply:
yes